### ISSUE TITLE
 file INSTALL cannot find "moneymanagerex/po/arabic.mo".

### ISSUE THREAD

**Issue by thomasboehm** [moneymanagerex/moneymanagerex#2262]
**Describe the bug**

build fails at the last step in branch 1.3.x

**To Reproduce**
1. run `cmake --build . --target package`
2. at the end you get the following error
```
...
[100%] Built target mmex
Run CPack packaging tool...
CPack: Create package using DEB
CPack: Install projects
CPack: - Run preinstall target for: MoneyManagerEx
CPack: - Install project: MoneyManagerEx
CMake Error at /home/thomas/src/moneymanagerex/build/po/cmake_install.cmake:41 (file):
  file INSTALL cannot find "/home/thomas/src/moneymanagerex/po/abkhazian.mo".
Call Stack (most recent call first):
  /home/thomas/src/moneymanagerex/build/cmake_install.cmake:81 (include)


CMake Error at /home/thomas/src/moneymanagerex/build/po/cmake_install.cmake:45 (file):
  file INSTALL cannot find "/home/thomas/src/moneymanagerex/po/albanian.mo".
Call Stack (most recent call first):
  /home/thomas/src/moneymanagerex/build/cmake_install.cmake:81 (include)


CMake Error at /home/thomas/src/moneymanagerex/build/po/cmake_install.cmake:49 (file):
  file INSTALL cannot find "/home/thomas/src/moneymanagerex/po/arabic.mo".
Call Stack (most recent call first):
  /home/thomas/src/moneymanagerex/build/cmake_install.cmake:81 (include)
...
```

**Expected behavior**
building should finish without error

**Additional context**

When I create a symlink *.po for each file *.mo in moneymanagerex/po/ the build finishes without error.

**Your OS:**
- [ ] Windows
- [ ] Mac OSX
- [x] Linux 
- [ ] Android -> please report bugs to moneymanagerex/android-money-manager-ex repo!

**MMEX version:**
- [ ] 1.4.x
- [x] 1.3.x
- [ ] 1.2.x
- [ ] 1.1 or older
- [ ] development version (git master branch)

**Comment by thomasboehm** [comment]
Why has this been tagged "outdated"? As far as I can see 1.3.x is the latest STABLE release. I'm aware of that there is a 1.4.x version in the master branch, but it is ALPHA (not even BETA). Many many people including me have problems converting the database from 1.3.x to 1.4.x. So there is no alternative to still using 1.3.x for a lot of people.

**Comment by vomikan** [comment]
That mean I want not to fix it in 1.3.x branch.
But in 1.4.0 this issue is not actual.
